manifests/install.pp

```diff
@@ -11,7 +11,11 @@
 # Copyright 2013 Patrick Mooney.
 # Copyright (c) IN2P3 Computing Centre, IN2P3, CNRS
 #
-class mit_krb5::install($packages = undef) {
+# @param packages
+#
+class mit_krb5::install (
+  Optional[String] $packages = undef,
```

Comment:
Should be `Variant[String, Array[String]]` if we're keeping the same logic already in place.